templates/mon_mysql.sql.j2

```diff
@@ -6,7 +6,7 @@ SET foreign_key_checks = 0;
 CREATE TABLE `alarm` (
   `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
   `alarm_definition_id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
-  `state` enum('UNDETERMINED','OK','ALARM') COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
Hibernate does not offer portable solution to map such column in the model. I mean, there's a way to do that but it works with MySQL only. We are trying to offer as portable Hibernate's model as possible but it wouldn't be possible if enums would be kept here. 

For reasons I cannot understand MySQL engines do not support CHECK clause we tried to replace enum with thus triggers were only reliable alternative to force constraints you had.  

The advantage is that schema in the form we pushed with this pull-request has better portability.